x-pack/plugins/screenshotting/server/screenshots/observable.ts

```diff
@@ -181,13 +177,14 @@ export class ScreenshotObservableHandler {
 
     return defer(() => getNumberOfItems(driver, this.logger, waitTimeout, this.layout)).pipe(
       mergeMap((itemsCount) => {
-        // set the viewport to the dimentions from the job, to allow elements to flow into the expected layout
+        // set the viewport to the dimensions from the job, to allow elements to flow into the expected layout
         const viewport = this.layout.getViewport(itemsCount) || getDefaultViewPort();
 
-        return forkJoin([
-          driver.setViewport(viewport, this.logger),
-          waitForVisualizations(driver, this.logger, waitTimeout, itemsCount, this.layout),
-        ]);
+        return driver
+          .setViewport(viewport, this.logger)
+          .then(() =>
+            waitForVisualizations(driver, this.logger, waitTimeout, itemsCount, this.layout)
+          );
```

Comment:
The `mergeMap` supports promises so we can revert it to the async-await code.

```suggestion
        await driver.setViewport(viewport, this.logger)
        await waitForVisualizations(driver, this.logger, waitTimeout, itemsCount, this.layout);
```